apps/src/code-studio/pd/workshop_enrollment/workshop_details.jsx

```diff
@@ -53,15 +53,22 @@ export default class WorkshopDetails extends React.Component {
   }
 
   location() {
-    return (
+    return this.props.workshop.virtual ? null : (
```

Comment:
Ack I'm still not understanding this! We have a check here that returns nothing if the workshop is virtual, but a check in the returned HTML when virtual = false for whether to show "Link to join" or "Location"?

Reply:
oh, good catch, i forgot to change that too